Day-2-Exercises/index.js

```diff
@@ -0,0 +1,102 @@
+let sentence = 'I hope this course is not full of jargon.'
+console.log(sentence.includes('jargon'))
+
+//Generate a random number between 0 and 100 inclusively.
+console.log(Math.floor(Math.random()*100))
+
+//Generate a random number between 50 and 100 inclusively.
+console.log(50 + Math.floor(Math.random()*50))
+
+//Generate a random number between 0 and 255 inclusively.
+console.log(Math.floor(Math.random()*255))
+
+//Access the 'JavaScript' string characters using a random number.
+let javaString = 'JavaScript'
+console.log(javaString.charAt(Math.random()*(javaString.length - 1)))
+
+//Use console.log() and escape characters to print the following pattern
+console.log('1 1 1 1 1\n2 1 2 4 8\n3 1 3 9 27\n4 1 4 16 64\n5 1 5 25 125')
+
+//Use substr to slice out the phrase because because because from the following sentence:'You cannot end a sentence with because because because is a conjunction'
+console.log(becauseCase.substring(becauseCase.indexOf('because'), becauseCase.lastIndexOf('because') + 'because'.length))
+
+
+//Exercise: Level 3
+
+//'Love is the best thing in this world. Some found their love and some are still looking for their love.' Count the number of word love in this sentence.
+let loveSentance = 'Love is the best thing in this world. Some found their love and some are still looking for their love.'
+let lovePatternCaseSensitive = /love/g
+let lovePatternCaseInSensitive = /love/gi
+
+console.log('Case sensitive result:',loveSentance.match(lovePatternCaseSensitive).length)
+console.log('Case insensitive result:',loveSentance.match(lovePatternCaseInSensitive).length)
+
+//Use match() to count the number of all because in the following sentence:'You cannot end a sentence with because because because is a conjunction'
+console.log('The ni=umber of \'because\' words in this sentance is',becauseCase.match(/because/g).length,)
+
+//Clean the following text and find the most frequent word (hint, use replace and regular expressions).
+const badSentence = '%I $am@% a %tea@cher%, &and& I lo%#ve %te@a@ching%;. The@re $is no@th@ing; &as& mo@re rewarding as educa@ting &and& @emp%o@weri@ng peo@ple. ;I found tea@ching m%o@re interesting tha@n any ot#her %jo@bs. %Do@es thi%s mo@tiv#ate yo@u to be a tea@cher!? %Th#is 30#Days&OfJavaScript &is al@so $the $resu@lt of &love& of tea&ching'
+let cleanPattern = /[%@$&;#]/g
+
+console.log(badSentence.replace(cleanPattern,''))
+
+
+
+//Calculate the total annual income of the person by extracting the numbers from the following text. 'He earns 5000 euro from salary per month, 10000 euro annual bonus, 15000 euro online courses per month.'
+let incomeText = 'He earns 5000 euro from salary per month, 10000 euro annual bonus, 15000 euro online courses per month.'
+let result = 0
+let incomeArray = incomeText.split(" ")
+console.log('Annual income: ', parseInt(incomeArray[2]) * 12 + parseInt(incomeArray[8]) + parseInt(incomeArray[12]) * 12)
```

Comment:
Good job. Can you do it using RegExp (regular expression) to extract numbers from the text?

Reply:
Added as second option to calculate